src/runtime/tests/recipe-test.ts

```diff
@@ -90,9 +91,9 @@ describe('recipe', () => {
           inMys: reads BigCollection<MyType>
     `);
 
-    const myType = manifest.findSchemaByName('MyType').entityClass()['type'];
-    const mySubType = manifest.findSchemaByName('MySubType').entityClass()['type'];
-    const otherType = manifest.findSchemaByName('OtherType').entityClass()['type'];
+    const myType = Entity.createEntityClass(manifest.findSchemaByName('MyType'), null)['type'];
```

Comment:
Is `.type` available? I think we've used it elsewhere.

Reply:
Um, Josh, that's the method I'm trying to get rid of.